test/onnx/test_pytorch_onnx_no_runtime.py

```diff
@@ -625,7 +625,9 @@ def forward(self, x, y):
         def symbolic_custom_invalid_add(g, input, other, alpha=None):
             return g.op("Add", input, other, invalid_attr_i=1)
 
-        torch.onnx.register_custom_op_symbolic("::add", symbolic_custom_invalid_add, 1)
+        torch.onnx.register_custom_op_symbolic(
+            "::add", symbolic_custom_invalid_add, opset_version=9
```

Comment:
Question for my understanding. Does this test still work even if without this change? That is, the custom symbolic function will still be invoked when registering for 1, exporting for 9?

Reply:
Since `add` is an implemented op in torch.onnx, registering a custom op to version 1 will not cause it to be selected with this change. If an op is not implemented then registering to version 1 will allow the exporter to pick it up.